### ISSUE TITLE
Arrangements of sequence diagrams

### ISSUE THREAD

**Issue by taneresme** [swe-ms-boun/2018fall-swe574-g1#80]
In the classroom meeting, we got the feedback from the lecturer and we will have to do some modifications in sequence diagrams accordingly. If you want we can share the tasks of modifications among us. What do you think?

**Comment by alidursen** [comment]
Can't we leave it to our meeting this saturday?

**Comment by vedat-akses** [comment]
Yes, let's start with it on Saturday and then continue to implementing features.
